dogs.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+    <title>Adoptadog</title>
+  </head>
+
+  <body>
+    <header class="header">
+      <div class="checkout">
+        <a href="checkout.html"><img src="images/cart-icon.svg" alt="cart icon"></a>
+        <p>$1234.56</p>
+        <input type="button" value="Checkout">
+      </div>
+      <img src="images/adoptadog-logo.svg" alt="adopt a dog logo">
+      <h1>Seattle's Premiere dog adoption agency</h1>
+      </header>
+
+      <nav class="nav">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+          </ul>
+        </nav>
+
+        <h2>Here are all our dogs!</h2>
```

Comment:
I would place this inside the container so it has the same margin as other elements